Lib/pprint.py

```diff
@@ -233,6 +233,34 @@ def _pprint_ordered_dict(self, object, stream, indent, allowance, context, level
 
     _dispatch[_collections.OrderedDict.__repr__] = _pprint_ordered_dict
 
+    def _pprint_dict_view(self, object, stream, indent, allowance, context, level, items=False):
+        key = _safe_tuple if items else _safe_key
+        write = stream.write
+        write(object.__class__.__name__ + '([')
+        if self._indent_per_level > 1:
+            write((self._indent_per_level - 1) * ' ')
+        length = len(object)
+        if length:
+            if self._sort_dicts:
+                entries = sorted(object, key=key)
+            else:
+                entries = object
+            self._format_items(entries, stream, indent, allowance + 1,
+                                    context, level)
+        write('])')
+
+    dict_keys_view = type({}.keys())
```

Comment:
```suggestion
    _dict_keys_view = type({}.keys())
```
Best to start these names with a leading underscore, or strange things will happen such as them showing up in autocomplete suggestions in IDEs

Reply:
Thanks, done!

Reply:
Thanks!